seqr/views/apis/summary_data_api.py

```diff
@@ -10,19 +11,22 @@
 from seqr.views.utils.orm_to_json_utils import _get_json_for_individuals, get_json_for_saved_variants_with_tags, \
     get_json_for_variant_functional_data_tag_types, get_json_for_projects, _get_json_for_families, \
     get_json_for_locus_lists, _get_json_for_models, get_json_for_matchmaker_submissions
-from seqr.views.utils.permissions_utils import analyst_required
+from seqr.views.utils.permissions_utils import analyst_required, user_is_analyst
 from seqr.views.utils.variant_utils import saved_variant_genes
-from settings import ANALYST_PROJECT_CATEGORY
+from settings import ANALYST_PROJECT_CATEGORY, API_LOGIN_REQUIRED_URL
 
 logger = logging.getLogger(__name__)
 
 MAX_SAVED_VARIANTS = 10000
 
 
-@analyst_required
+@login_required(login_url=API_LOGIN_REQUIRED_URL)
 def mme_details(request):
-    submissions = MatchmakerSubmission.objects.filter(
-        deleted_date__isnull=True, individual__family__project__projectcategory__name=ANALYST_PROJECT_CATEGORY)
+    submissions = MatchmakerSubmission.objects.filter(deleted_date__isnull=True)
+    if user_is_analyst(request.user):
+        submissions = submissions.filter(individual__family__project__projectcategory__name=ANALYST_PROJECT_CATEGORY)
+    else:
+        submissions = submissions.filter(individual__family__project__can_view_group__user=request.user)
```

Comment:
Should we also check the user has permissions for the project on AnVIL?

Reply:
oh yes, great catch!